.github/pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+### Github Issue
+<!--
+    Add the github issue number if exists(ex. Fixes #1, Closes #1 or Resolves #1).
+-->
```

Comment:
Can we remove those `<!--` ?
I'm not sure XML comments make sense in this context.
Also:

```
Add the Github issue number if one exists, prefixed by one of Github's keywords, ex. `Fixes #1`, `Closes #1` or `Resolves #1`.
```